### PR TITLE
Dockerfile 4 dev prod

### PR DESCRIPTION
**Pull request by laipogo**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior?** (You can also refer to a JIRA ticket here)

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

* **What dependencies / modules need to be updated as pre-requisites?** (Include relevant PR links here)

* **Description of changes**

---

* **Please check if the PR fulfills these requirements**
- [ ] The subject of this PR contains a JIRA ticket BTFS-xxxx (N/A for community PRs)
- [ ] PR description and commit messages are [good and meaningful](https://chris.beams.io/posts/git-commit/)
- [ ] Unit tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

* **Please make sure the following procedures have been applied before requesting reviewers**
- [ ] Code changes closely follow [Go Code Review Comments](https://github.com/golang/go/wiki/CodeReviewComments)
- [ ] Code has been *rebased* against latest master (`git merge` not recommended, unless you know what you are doing)
- [ ] Code changes have run through `go fmt`
- [ ] Code changes have run through `go mod tidy`
- [ ] All unit tests passed locally (`make test_go_test`)

### CODE REVIEW

File: Dockerfile.prod [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+
+# The default logging level
+ENV BTFS_LOGGING ""
+
+# This just makes sure that:
+# 1. There's an fs-repo, and initializes one if there isn't.
+# 2. The API and Gateway are accessible from outside the container.
+ENTRYPOINT ["/sbin/tini", "--", "/usr/local/bin/start_btfs"]
+
+RUN btfs init
```
Comment (taiyangc) [inline Dockerfile.prod]: can you add `RUN btfs config profile apply storage-client` in case in the future there are additional steps

Comment (laipogo) [reply Dockerfile.prod]: tested

File: bin/container_daemon_prod [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  echo "Found BTFS fs-repo at $repo"
+else
+  case "$IPFS_PROFILE" in
+    "") INIT_ARGS="" ;;
+    *) INIT_ARGS="--profile=$IPFS_PROFILE" ;;
+  esac
+
+  btfs init $INIT_ARGS
+  btfs config profile apply storage-client
+  sed -i "" 's/\"Analytics\": false,/\"Analytics\": true,/g' $repo/config
```
Comment (taiyangc) [inline bin/container_daemon_prod]: `btfs config Experimental.Analytics true`

:)

Comment (laipogo) [reply bin/container_daemon_prod]: i c,

**Pull request merged**